client/me/help/help-contact/index.jsx

```diff
@@ -260,7 +260,14 @@ module.exports = React.createClass( {
 		}
 
 		if ( ! ( olark.isOlarkReady && sitesInitialized ) ) {
-			return <div className="help-contact__placeholder" />;
+			return (
+				<div className="help-contact__placeholder">
+					<HelpContactForm disabled={true}
+						showSubjectField={ false }
+						onSubmit={ function() {} }
```

Comment:
You can use lodash `noop` here if using an empty function makes this feel hacky to you.